### ISSUE TITLE
probabilistic weight for each labelling function?

### ISSUE THREAD

**Issue by arfeen93** [snorkel-team/snorkel#1714]
Does Snorkel have probabilistic weight for each labelling function accounting for predicted label ? For example, suppose we have three labeling function resulting in a particular label output as 1. Does snorkel provide the weights corresponding to each labelling function resulting in a particular prediction? If yes, Can you please point out to the correct direction.

**Comment by yinxiangshi** [comment]
Yes, if you just wanna see the prob output of the snorkel, you can use:

`snorkel.labeling.model.label_model.LabelModel.get_weights()`

Doc is here: https://snorkel.readthedocs.io/en/v0.9.7/packages/_autosummary/labeling/snorkel.labeling.model.label_model.LabelModel.html?highlight=state_dict#snorkel.labeling.model.label_model.LabelModel.get_weights
